service/src/test/resources/db/migration/beforeMigrate.sql

```diff
@@ -1,9 +0,0 @@
-/***********************************************************************************************************************
```

Comment:
I was just about to suggest that whatever's put in the V1 script to alter the databases could've also been put here, but there's no real difference.